staging/src/k8s.io/code-generator/pkg/util/normalizers.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import "strings"
+
+const EmptyString = ""
+const Dash = "-"
```

Comment:
do these have to be public?

Reply:
Hye, The idea was to reuse it after to avoid "Magic Value".

Reply:
what is more magic about `"-"` than `Dash`? I can see multi-char strings to be valuable, but not these.

Reply:
ok, i will removed it. Do i have to change Empty String const or not ?